docs/source/user_guide/k8s.rst

```diff
@@ -33,7 +33,7 @@ foo.yml
 
 kubeitpod.yml
 
-Note at this time it is necessary to have either a tcp-listener or a tcp-peer to be able to start the control service. See https://github.com/ansible/receptor/issues/518
+N.B. it may necessary to have one of tcp-listener, tcp-peer, or local-only to be able to start the control service. See https://github.com/ansible/receptor/issues/518
```

Comment:
We shouldn't use N.B. here.  You can turn this into an RST note directive to get better highlighting. See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Reply:
Agreed, changed to note